client/src/pages/ResetPasswordModal.tsx

```diff
@@ -0,0 +1,196 @@
+import { OverlayTrigger, Popover } from "react-bootstrap";
+import React, { FunctionComponent, useState } from "react";
+import { Redirect } from "react-router-dom";
+
+import ConfirmationModal from "./ConfirmationModal";
+import { handlePasswordReset } from "../services/auth";
+import LogoModal from "../components/organisms/LogoModal";
+import { TextField } from "../components/atoms/TextField";
+
+const ResetPasswordModal: FunctionComponent = () => {
+  const [password, setPassword] = useState(""); // can we put these all in an array somehow?????
+  const [hidePassword, setHidePassword] = useState(true);
+  const [hasOneLowerCase, setHasOneLowerCase] = useState(false);
+  const [hasOneUpperCase, setHasOneUpperCase] = useState(false);
+  const [hasOneNumber, setHasOneNumber] = useState(false);
+  const [hasOneSymbol, setHasOneSymbol] = useState(false);
+  const [hasTwelveCharacterMin, setHasTwelveCharacterMin] = useState(false);
```

Comment:
Are we actually relying on these in any way? From what I can tell, the `forEach` loop inside `onChangePass` updates the state but ends up setting the requirements within the function without needed the individual states. To me, looks like everything works even if the following lines (inclusive) are deleted: 13-17, 60-66, 83. 

Also, ignore the other comment I made if this is true.

Reply:
Yup! ended up deleting those lines :)